geometry_msgs/Vector3 dir1
geometry_msgs/Vector3 dir2
geometry_msgs/Vector3 dir3
geometry_msgs/Vector3 dir4